src/_sass/index.scss

```diff
@@ -1,8 +1,7 @@
 @charset "utf-8";
 
-@import "../../submodules/ca-state-template/source/scss/cagov.core.scss";
-@import "../../submodules/ca-state-template/source/scss/colorscheme/colorscheme-oceanside-import.scss";
 @import "variables";
+@import "breakpoints";
 
 @import "../../submodules/tabulator/src/scss/bootstrap/tabulator_bootstrap4.scss";
```

Comment:
I think this is the only other reason we have a submodule. If we can get rid of this import, I think we can get rid of that other submodule as well.

Reply:
I attempted to remove this submodule and replace it with the CDN links, but it breaks the CSS _so_ much. I would rather not remove it at the time being.

Reply:
Yikes! Sorry about that. I guess we are relying on it more than I thought...